### PR TITLE
added test for FormFactoryCompilerPass

### PR DESCRIPTION
**Pull request by ahmetakbn**
### Subject

Improving tests for `SonataCoreBundle`

### CODE REVIEW

File: Tests/DependencyInjection/Compiler/FormFactoryCompilerPassTest.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+use Sonata\CoreBundle\DependencyInjection\Compiler\FormFactoryCompilerPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+/**
+ * @author Ahmet Akbana <ahmetakbana@gmail.com>
+ */
+final class FormFactoryCompilerPassTest extends AbstractCompilerPassTestCase
+{
+    public function registerCompilerPass(ContainerBuilder $container)
```
Comment (Soullivaneuh) [inline Tests/DependencyInjection/Compiler/FormFactoryCompilerPassTest.php]: Could you add a `inheritdoc` phpdoc tag for this method?

Thanks.

Comment (ahmetakbn) [inline Tests/DependencyInjection/Compiler/FormFactoryCompilerPassTest.php]: done

**Comment by Soullivaneuh** [comment]
RTM, thank @ahmetakbn!

**Comment by greg0ire** [comment]
Thanks @ahmetakbn !

**Pull request merged**